pkg/output/table.go

```diff
@@ -247,3 +247,12 @@ func (t *Table) isMap() bool {
 	_, ok := t.objects[0].(map[string]string)
 	return ok
 }
+
+func isZero(value reflect.Value) bool {
+	// special case: value.IsZero() doesn't evaluate to true for maps that are empty but not nil
```

Comment:
What about slices that are empty but not `nil`?

Reply:
It probably has the same issue, but I didn't investigate

Reply:
It did have the same problem. I updated the code to handle both.